packages/libs/react-ui/src/components/Notification/NotificationContainer.tsx

```diff
@@ -17,7 +19,7 @@ export interface INotificationProps {
   title?: string;
   children?: React.ReactNode;
   expanded?: boolean;
-  color?: keyof typeof cardColorVariants;
+  color?: NotificationColors;
```

Comment:
nit: I have a preference to keep it how it was for the sake of defining less types